traversal/procedure/CombinationProcedure.java

```diff
@@ -150,60 +146,14 @@ private void registerBFS(StructureVertex.Type start) {
 
     private ProcedureEdge<?, ?> createOut(ProcedureVertex.Type from, ProcedureVertex.Type to, StructureEdge<?, ?> structureEdge,
                                           int order) {
-        ProcedureEdge<?, ?> edge;
-        if (structureEdge.isNative()) {
-            switch (structureEdge.asNative().encoding().asType()) {
-                case SUB:
-                    edge = new ProcedureEdge.Native.Type.Sub.Forward(from, to, order, structureEdge.asNative().isTransitive());
-                    break;
-                case OWNS:
-                    edge = new ProcedureEdge.Native.Type.Owns.Forward(from, to, order, false);
-                    break;
-                case OWNS_KEY:
-                    edge = new ProcedureEdge.Native.Type.Owns.Forward(from, to, order, true);
-                    break;
-                case PLAYS:
-                    edge = new ProcedureEdge.Native.Type.Plays.Forward(from, to, order);
-                    break;
-                case RELATES:
-                    edge = new ProcedureEdge.Native.Type.Relates.Forward(from, to, order);
-                    break;
-                default:
-                    throw TypeDBException.of(UNRECOGNISED_VALUE);
-            }
-        } else if (structureEdge.isEqual()) {
-            return new ProcedureEdge.Equal(from, to, order, Encoding.Direction.Edge.FORWARD);
-        } else throw TypeDBException.of(ILLEGAL_STATE);
```

Comment:
this implementation to create `Procedure` objects from `Structure` directly, skipping `Planner` has been embedded in static methods in `Procedure`